client/session.go

```diff
@@ -1146,7 +1143,7 @@ func (s *session) streamBlocksBatchFromPeer(
 					xlog.NewLogField("actualStart", block.Start),
 					xlog.NewLogField("indexID", i),
 					xlog.NewLogField("indexBlock", j),
-				).Errorf("stream blocks response from peer %v returned mismatched ID", peer.Host().String())
+				).Errorf("stream blocks response from peer %s returned mismatched block start", peer.Host().String())
```

Comment:
For this error msg and the ones above, would it be useful for debugging to add more details? Ie, here insert the expected block start and actual block start so that we know it's always off by X mins etc?